docs/guides/private-submodules.rst

```diff
@@ -11,42 +11,21 @@ In order to give Read the Docs access to clone your submodules you'll need to ad
 
 .. note::
 
-   You can manage which submodules Read the Docs should clone using a configuration file.
-   See :ref:`config-file/v2:submodules`.
+   - You can manage which submodules Read the Docs should clone using a configuration file.
+     See :ref:`config-file/v2:submodules`.
 
-.. note::
-
-   Make sure you are using ``SSH`` URLs for your submodules
-   (``git@github.com:readthedocs/readthedocs.org.git`` for example)
-   in your ``.gitmodules`` file, not ``http`` URLs.
+   - Make sure you are using ``SSH`` URLs for your submodules
+     (``git@github.com:readthedocs/readthedocs.org.git`` for example)
+     in your ``.gitmodules`` file, not ``http`` URLs.
```

Comment:
nit: I think having 2 different notes is more visible and easier to parse while reading.